DHFPROD-8636: E2E tests for validating all the widgets in Twizzlers

ngodugu-marklogic opened the pull request:
### Description

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests
- [ ] Added to Release Wiki

wooldridge reviewed (changes requested):
I'm seeing some failures on my run with a clean install.

<img width="812" alt="Screen Shot 2022-04-25 at 10 35 25 PM" src="https://user-images.githubusercontent.com/477757/165229139-2368cdfa-fffe-4f33-9a43-75cd98ab52ee.png">


Attaching some of the terminal output:
[widgetValidations.spec.tsx,txt](https://github.com/marklogic/marklogic-data-hub/files/8560158/widgetValidations.spec.tsx.txt)

ngodugu-marklogic commented:
> I'm seeing some failures on my run with a clean install.
> 
> <img alt="Screen Shot 2022-04-25 at 10 35 25 PM" width="812" src="https://user-images.githubusercontent.com/477757/165229139-2368cdfa-fffe-4f33-9a43-75cd98ab52ee.png">
> 
> Attaching some of the terminal output: [widgetValidations.spec.tsx,txt](https://github.com/marklogic/marklogic-data-hub/files/8560158/widgetValidations.spec.tsx.txt)

<img width="726" alt="image" src="https://user-images.githubusercontent.com/74939216/165379829-712c997a-d274-43c0-af70-7fd418cdc8f1.png">
@wooldridge Seems like the indexes of the widgets got changed since we added created and updated widgets recently. Updated the PR accordingly and the tests went fine on clean build. PR is ready for review
